src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -287,6 +282,17 @@ private void RemovePlaceholderReplacement(BoundValuePlaceholderBase placeholder)
             Debug.Assert(removed);
         }
 
+        /// <summary>
+        /// Remove all the listed placeholders.
+        /// </summary>
+        private void RemovePlaceholderReplacements(IEnumerable<BoundValuePlaceholderBase> placeholders)
```

Comment:
Could we use more specific type rather than IEnumerable<BoundValuePlaceholderBase>? ArrayBuilder/ImmutableArray